http-client-spi/src/main/java/software/amazon/awssdk/http/DefaultSdkHttpFullRequest.java

```diff
@@ -149,6 +150,19 @@ public SdkHttpMethod method() {
                 .content(content);
     }
 
+    @Override
+    public String toString() {
```

Comment:
Is this something we did in V1? Is there potential for sensitive headers/query params to be logged?

Reply:
Yes, that's what we do in v1. I don't think there are any sensitive information in  the headers or query params and those info can be easily captured anyway.

https://github.com/aws/aws-sdk-java/blob/46c00efdd9240d2d20d734f806dcc71506af6e0f/aws-java-sdk-core/src/main/java/com/amazonaws/DefaultRequest.java#L300-L311

Reply:
yeah, makes sense and now I remember I've seen those requests. I'll remove the query parameters and headers. 

We are currently logging those in the `Aws4Signer`, probably should remove the logging there too?
https://github.com/aws/aws-sdk-java-v2/blob/8f89e2d772f0e77400939ef3bd524e27a4f65c08/core/auth/src/main/java/software/amazon/awssdk/auth/signer/internal/AbstractAws4Signer.java#L176-L194

Reply:
Would it be beneficial to log which headers and query string parameters are present, even if we don't log their values? That's what I do in the profile's `toString`: show which properties are set, but not their values.

Reply:
+1, sounds good to me. will add the logging for the header and query parameter keys.